examples/cli.js

```diff
@@ -9,19 +9,18 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
-let cookies = `NID=; SID=; __Secure-1PSID=; __Secure-3PSID=; HSID=; SSID=; APISID=; SAPISID=; __Secure-1PAPISID=; __Secure-3PAPISID=; SIDCC=; __Secure-1PSIDCC=; __Secure-3PSIDCC=`;
-
-let bot = new Bard(cookies, {
-  proxy: {  // optional
-    host: process.env.PROXY_HOST,
-    port: process.env.PROXY_PORT,
-    auth: {
-      username: process.env.PROXY_USERNAME,
-      password: process.env.PROXY_PASSWORD
-    },
-    protocol: process.env.PROXY_PROTOCOL
-  }
-});
+let cookies = `__Secure-1PSID=`;
+
+let yourConversationObject = `{"conversationId":"c_93c9fbe53c884882","requestId":"r_93c9fbe53c884b3b","responseId":"rc_93c9fbe53c884279","responses":["It is a pleasure to meet you, Eris. I am Bard.","Eris","It is a pleasure to meet you, Eris. I am Bard.","It is a pleasure to meet you, Eris! I am Bard. What can I help you with today?","Hello Eris! It's a pleasure to meet you. I'm Bard."]}`
```

Comment:
why it's defined as a string and then parsed?

Reply:
I find it easier since then you can just copy the output from the terminal (Which logs the object) and paste it in there.